<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Экспорт</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>4</number>
     </property>
     <property name="verticalSpacing">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="clientLabel">
       <property name="text">
        <string>Игровой клиент</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="clientCombo"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="versionCombo"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="versionLabel">
       <property name="text">
        <string>Версия</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="directoryLabel">
       <property name="text">
        <string>Директория</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLineEdit" name="dirEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="dirlButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="log">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>Для копирования файлов необходимых только для указанной версии клиента в указанную директорию нажмите кнопку &quot;Экспортировать&quot;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <item>
      <spacer name="hspc">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>388</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Экспортировать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
